README.md

```diff
@@ -162,13 +163,17 @@ Verification inputs:
                         used with multiple inputs (default: None)
   --signature FILE      The signature to verify against; not used with
                         multiple inputs (default: None)
+  --rekor-bundle FILE   The offline Rekor bundle to verify with; not used with
+                        multiple inputs (default: None)
 
 Extended verification options:
   --cert-email EMAIL    The email address to check for in the certificate's
                         Subject Alternative Name (default: None)
   --cert-oidc-issuer URL
                         The OIDC issuer URL to check for in the certificate's
                         OIDC issuer extension (default: None)
+  --rekor-offline       Perform offline Rekor verification using a bundle;
+                        implied by --rekor-bundle (default: False)
```

Comment:
Why is the default false? Wouldn't it make sense that in this case:
1. `--rekor-bundle` is not set 
2. but `FILE.bundle` is found

you would use offline verification -- this corresponds to automatically finding and using `FILE.sig` and `FILE.cert`.

Maybe the default should be rekor-offline=TRUE if there is a bundle (that is if `--rekor-bundle` is set or `FILE.bundle` is found). Otherwise rekor-offline should be FALSE. Regardless of the files existence, user should be able to override to either direction: offline or online)

Reply:
> you would use offline verification -- this corresponds to automatically finding and using FILE.sig and FILE.cert if they are not specified.

That's the current default! I can see how the flag's name is a little confusing: what it actually does is *force* offline Rekor verification, and fails if it's impossible. By default, offline Rekor verification is opportunistic. 

In other words, here's the current set of behaviors:

* `sigstore verify foo.txt`: look for `foo.txt.{crt,sig,bundle}` and use all three if they exist, otherwise fall back on online Rekor verification
* `sigstore verify --rekor-offline foo.txt`: look for `foo.txt.{crt,sig,bundle}` and fail unless all three exist

Reply:
In other words, maybe the right flag name is `--force-rekor-offline` or similar, to emphasize that this doesn't affect the "opportunistic" behavior. CC @asraa and @di for thoughts.

Reply:
thanks -- I should have checked the code before complaining.

I think the name might be fine, it's just the docstring that could maybe be tweaked, maybe `...implied by --rekor-bundle or the existence of FILE.bundle` (although that is not super clear either)

Reply:
Maybe I could change it from "perform" to "require", i.e.:

```
  --rekor-offline       Require offline Rekor verification using a bundle;
                        implied by --rekor-bundle (default: False)
```

That emphasizes that `--rekor-offline` forces bundle use, but doesn't imply that other modes won't do so opportunistically.

Reply:
The doc changes lgtm, maybe `--require-rekor-offline`?

Reply:
Sounds good to me!